Table Creation

Signior-X opened the pull request:
I have created a folder Make_Tables which has the code to create tables in the local server.
One needs to change the username and password according to their system in the text file in that folder.
Also, moved the html file into the templates folder.
This code will also give general idea to others, how to add some data into the table also.

Signior-X committed:
Added createdatabase.py, .env and updated README.md

Signior-X commented:
One has to first edit the mysqlinfo.txt and add there database credentials and then run createdatabase.py file

Milind712000 reviewed (changes requested):
Task List : 
- [x] create a requirements.txt and add the dependencies used
- [x] Add the createdatabase step to Readme
- [x] Also edit the readme with instructions for installing dependencies from requirements.txt and the command to run the app `python3 app.py`
- [x] Remove the change made to basic.html, it'll be fixed in another issue

Milind712000 commented on Make_Tables/createdatabase.py:
```diff
@@ -0,0 +1,46 @@
+try:
+	mycursor.execute("drop database sac_data")
+except:
+	print("Database does exist already")
```
Fix the error message
database does not exist

Signior-X replied:
Done changes it and squashed the commits also.

Signior-X commented:
I have created the requirements.txt file and also added the description in README.md file, check the changes

Milind712000 reviewed (changes requested):
- [x] Instead of a raw text file switch to a .json or .env file to hold the credentials (you can use something like python-dotenv)
- [x] Add the foreign key constraints
- [x] Rename LoginDetails to Users
- [x] Rename ClubDetails to Clubs
- [x] Rename groupid and groupname to clubID and clubName
- [x] Add newline at the end of createdatabase.py

Apart from these the database and tables look good
and the changes to Readme and requirements.txt are also good

Signior-X commented:
I have done the changes as you said, created the .env file and used it using python-dotenv, and also added it in the requirements.txt file.
I have added the Foreign key constraints in the Coordies and Registered Table, Also changed the name of the tables.
Rest also done, check them.

Milind712000 reviewed (changes requested):
- [x] move .env to the root of the project as our flask app will also use it 
- [x] add the events table
- [x] remove the dummy data which is being inserted
- [x] Rename Coordies to Coordinators
- [x] (__optional__) Rename Registered to something better

Signior-X commented:
Changed Registered to ClubMembers
Added the Events Table
Moved .env
done rest all, review them

Milind712000 commented:
Fixes #6
